MarsQA-1/SpecflowPages/Pages/SkillsPage.cs

```diff
@@ -0,0 +1,304 @@
+
+            AddBtn.Click();
+        }
+
+        public static void VerifySkillsRecords()
+        {
+            Thread.Sleep(2000);
+            Assert.AreEqual(ExcelLibHelper.ReadData(2, "Skill"), SkillLbl.Text, "First skill has not been added");
+            Assert.AreEqual(ExcelLibHelper.ReadData(2, "Level"), SkillLevelLbl.Text, "First skill level has not been added");
+            Assert.AreEqual(ExcelLibHelper.ReadData(3, "Skill"), SkillLbl2.Text, "Second skill has not been added");
+            Assert.AreEqual(ExcelLibHelper.ReadData(3, "Level"), SkillLevelLbl2.Text, "Second skill level has not been added");
+            Assert.AreEqual(ExcelLibHelper.ReadData(4, "Skill"), SkillLbl3.Text, "Third skill has not been added");
+            Assert.AreEqual(ExcelLibHelper.ReadData(4, "Level"), SkillLevelLbl3.Text, "Third skill level has not been added");
+        }
+
+        public static void AddNewSkillWithSpecialCharacters()
+        {
+            Thread.Sleep(3000);
+            AddNewBtn.Click();
+            AddSkillTxt.SendKeys("SQL!@#");
+
+            //create select element object 
+            var selectElement = new SelectElement(SkillLevelDdn);
+
+            //select by value
+            selectElement.SelectByValue("Beginner");
+
+            AddBtn.Click();
+        }
+
+        public static void VerifyNewSkillWithSpecialCharacters()
+        {
+            Thread.Sleep(2000);
+            Assert.That(PopUpMsg.Text == "SQL!@# has been added to your skills", "skill has not been added");
+            Assert.AreEqual("SQL!@#", SkillLbl.Text, "skill has not been added");
+            Assert.AreEqual("Beginner", SkillLevelLbl.Text, "skill level has not been added");
+        }
+
+        public static void AddNewSkillWith100Characters()
+        {
+            Thread.Sleep(3000);
+            AddNewBtn.Click();
+            AddSkillTxt.SendKeys("QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM12");
+
+            //create select element object 
+            var selectElement = new SelectElement(SkillLevelDdn);
+
+            //select by value
+            selectElement.SelectByValue("Intermediate");
+
+            AddBtn.Click();
+        }
+
+        public static void VerifyNewSkillWith100Characters()
+        {
+            Thread.Sleep(2000);
+            Assert.That(PopUpMsg.Text == "QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM12 has been added to your skills", "skill has not been added");
+            Assert.AreEqual("QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM1234567890QWERTYUIOPASDFGHJKLZXCVBNM12", SkillLbl.Text, "skill has not been added");
+            Assert.AreEqual("Intermediate", SkillLevelLbl.Text, "skill level has not been added");
```

Comment:
can you please move all your asserts to steps, good practice not to mix asserts with page logic, so keep asserts in steps

Reply:
or in assert helper classes which will be classed in the steps